### PR TITLE
Mtd 14 error handler

### PR DESCRIPTION
**Pull request by ghilyana**

### CODE REVIEW

File: src/main/java/com/ghilly/service/CountryService.java [hunk 1]
```diff
@@ -14,4 +14,6 @@ public interface CountryService {
     void update(Country country);
 
     void delete(int countryId);
+
+    void exists (int countryId);
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/service/CountryService.java]: Why is it public? You use the method only in CountryServiceRest

File: src/main/java/com/ghilly/service/CountryServiceRest.java [hunk 2]
```diff
@@ -41,26 +42,29 @@ public List<Country> getAllCountries() {
...
-        if (!repository.existsById(countryId)) {
-            throw new IllegalArgumentException("The country with the ID " + countryId + " is not found.");
-        }
+        exists(countryId);
         repository.deleteById(countryId);
         logger.info("The country with ID {} was deleted", countryId);
     }
+
+    @Override
+    public void exists(int countryId) {
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/service/CountryServiceRest.java]: why public?

**Review by gregorusanov — CHANGES_REQUESTED** [review]

File: src/main/java/com/ghilly/service/CountryService.java (hunk 1 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/service/CountryService.java]: Done

File: src/main/java/com/ghilly/service/CountryServiceRest.java (hunk 2 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/service/CountryServiceRest.java]: Done

**Review by gregorusanov — APPROVED** [review]

**Pull request merged**